pkg/cmd/rm.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/ClusterHQ/dvol/pkg/datalayer"
+	"github.com/spf13/cobra"
+)
+
+var forceRemoveVolume bool
+
+func userIsSure(extraMessage string) bool {
+	message := fmt.Sprintf("Are you sure? %s (y/n): ", extraMessage)
+	fmt.Print(message)
+	var response string
+	_, err := fmt.Scanln(&response)
+	if err != nil {
+		fmt.Println("Error reading response.")
+		return false
+	}
+	response = strings.ToLower(response)
+	if response == "y" || response == "yes" {
```

Comment:
Looks like this will return False if the user enters anything but 'yes/y', it should probably either re-prompt or just say 'Enter y/yes to confirm' or something like that